regression/contracts/assigns_enforce_statics_subfunctions/test.desc

```diff
@@ -0,0 +1,14 @@
+CORE
+main.c
+--enforce-contract bar _ --pointer-primitive-check
```

Comment:
Why do we need `--pointer-primitive-check` here?

Reply:
?

Reply:
Why is there an underscore here (after `bar`)?

Reply:
this underscore separates two groups of options , the first group is passed to goto-instrument to perform the contract instrumentation, and the second set is passed to cbmc to analyse the instrumented contracts.